vcd/resource_vcd_network.go

```diff
@@ -151,7 +155,7 @@ func resourceVcdNetwork() *schema.Resource {
 	}
 }
 
-func resourceVcdNetworkCreate(d *schema.ResourceData, meta interface{}) error {
+func resourceVcdNetworkDeprecatedCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Do we really need to mark deprecation of this function? Having "Deprecated" inside the name looks nasty. We should be deprecating the whole resource instead.

Can we make resource_vcd_network.go and the resource_vcd_network_routed.go, which replaces it, be functionally identical? Meaning that the old resource_vcd_network.go would just call the new resource_vcd_network_routed? In that case, even if someone were using the deprecated resource_vcd_network.go, it would be fine. We would only want to print a warning.

Reply:
Thank you. We don't like too much code :)